### DISCUSSION TITLE
Imtegrate mapbox-gl-compare

### DISCUSSION THREAD

**Discussion by piedmontlong** [uber/react-map-gl#1604] [Ideas]
Is it possible to integrate mapbox-gl-compare plugin?

**Comment by jcsho** [comment]
There's a POC for `mapbox-gl-compare` that I am trying to use here: https://codesandbox.io/s/po0dn?file=/src/App.js

However, it does not seem to work with the latest `6.x` version of `react-map-gl` with the error `Maximum update depth exceeded` occurring at the useCallback hook

Wondering if anyone found a workaround?
